chart/kubeapps/templates/frontend/configmap.yaml

```diff
@@ -103,47 +103,6 @@ data:
       }
     {{- end }}
 
-      # Forward '/api/assetsvc' to '/assetsvc'
-      # but preserving the encoding (eg. '%2F' is not converted to '/')
-      # see https://serverfault.com/a/906479
-      # Ex: from "/api/assetsvc/what$2Fever?param=value"
-      #     it matches as $1="/what$2Fever" and $args="param=value"
-      #     downstream services will receive "/assetsvc/what$2Fever?param=value"
-      location ~* /api/assetsvc {
```

Comment:
Although, in theory, it is safe to remove this piece of code, I wouldn't do it in a soon-to-be _patch_ release. What if we just render this excerpt if assetsvc.enabled=true ? And it is in the next _minor_ that we effectively remove it?

Reply:
Yeah true. I'd forgotten that people could potentially be using the current kubeops outside of our dashboard usage. +1 for just putting it behind a flag like you've done for assetsvc.